tests/behat/features/ra_management_grants.feature

```diff
@@ -0,0 +1,181 @@
+Feature: A RA(A) should only have access to certain pages
+
+  Scenario: Provision a institution and a user to promote later on by an authorized institution
+    Given institution "stepup.example.com" can "select_raa" from institution "stepup.example.com"
+    And institution "institution-a.example.com" can "use_ra" from institution "stepup.example.com"
+    And institution "institution-b.example.com" can "use_raa" from institution "stepup.example.com"
+    And institution "institution-d.example.com" can "select_raa" from institution "institution-d.example.com"
+    And institution "institution-d.example.com" can "use_raa" from institution "institution-d.example.com"
+    And a user "RA institution A" identified by "urn:collab:person:stepup.example.com:joe--ra" from institution "stepup.example.com" with UUID "00000000-0000-0000-0000-000000000001"
+    And a user "RAA institution B" identified by "urn:collab:person:stepup.example.com:joe--raa" from institution "stepup.example.com" with UUID "00000000-0000-0000-0000-000000000002"
+    And a user "RAA institution D" identified by "urn:collab:person:institution-d.example.com:joe-d-raa" from institution "institution-d.example.com" with UUID "00000000-0000-0000-0000-000000000003"
+    And a user "RA(A) candidate" identified by "urn:collab:person:institution-d.example.com:joe--candidate" from institution "stepup.example.com" with UUID "00000000-0000-0000-0000-000000000004"
+    And the user "urn:collab:person:stepup.example.com:joe--ra" has a vetted "yubikey"
+    And the user "urn:collab:person:stepup.example.com:joe--raa" has a vetted "yubikey"
+    And the user "urn:collab:person:institution-d.example.com:joe-d-raa" has a vetted "yubikey"
+    And the user "urn:collab:person:institution-d.example.com:joe--candidate" has a vetted "yubikey"
+    And the user "urn:collab:person:stepup.example.com:joe--ra" has the role "ra" for institution "stepup.example.com"
+    And the user "urn:collab:person:stepup.example.com:joe--raa" has the role "raa" for institution "stepup.example.com"
+    And the user "urn:collab:person:institution-d.example.com:joe-d-raa" has the role "raa" for institution "institution-d.example.com"
+
+
+  # Token page
```

Comment:
I did not expect Token page tests in this feature. Can you further explain the name of this feature? `ra_management_grants.feature`.

Reply:
Maybe choosing a feature name referring to testing authorizations is more on point.

Reply:
Removed the `management` part from the name.